partiql-spi/src/main/kotlin/org/partiql/spi/function/Builtins.kt

```diff
@@ -205,12 +205,10 @@ internal object Builtins {
         Fn_POS__DECIMAL_ARBITRARY__DECIMAL_ARBITRARY,
         Fn_POSITION__STRING_STRING__INT64,
         Fn_POSITION__CLOB_CLOB__INT64,
-
-        Fn_SUBSTRING__STRING_INT32__STRING,
-        Fn_SUBSTRING__STRING_INT32_INT32__STRING,
-        Fn_SUBSTRING__CLOB_INT64__CLOB,
-        Fn_SUBSTRING__CLOB_INT64_INT64__CLOB,
-
+        Fn_SUBSTRING_2STR,
+        Fn_SUBSTRING_2CLOB,
+        Fn_SUBSTRING_3STR,
+        Fn_SUBSTRING_3CLOB,
```

Comment:
any reason for these to be renamed? I think keeping things consistently named would be clearer